reference/key-concepts.mdx

```diff
@@ -113,6 +113,22 @@ Protocol buffers (commonly known as Protobuf) are Google's language-neutral, pla
 
 `psql` is a terminal-based front-end to PostgreSQL and other databases that are compatible with the PostgreSQL [wire protocol](#wire-protocol), such as RisingWave. With `psql`, you can type queries interactively, issue these queries to RisingWave, and see the query results. In addition, `psql` provides a number of meta-commands and various shell-like features to facilitate writing scripts and automating a wide variety of tasks.
 
+### `risectl`
+
+`risectl` is a command-line tool for managing the RisingWave kernel. It allows you to inspect the cluster status and access low-level APIs for cluster control. This tool is included with each [version release](https://github.com/risingwavelabs/risingwave/releases/). Please use it at your own risk. Run `risectl help` for more details.
+
+### `risedev`
+
+`risedev` is a command-line utility that facilitates the development of RisingWave. It is built on `cargo-make`, a crucial tool for building Rust projects. `risedev` is included in RisingWave's source code. Additionally, `risectl` is integrated into `risedev` as a subcommand: `risedev ctl`.
+
+### `risingwave`
+
+`risingwave` is the standalone binary for the RisingWave database. You can initiate different nodes using subcommands; for instance, `risingwave compute-node` launches the compute node. Run `risingwave help` for more details. `risectl` is also incorporated into `risingwave` as a subcommand, `risingwave ctl`. 
+
+### `rwc`
+
+`rwc` (RisingWave Cloud CLI) is a command-line tool for accessing and managing RisingWave Cloud/BYOC instances via open APIs. It is particularly useful for [setting up the BYOC cluster](/cloud/project-byoc#create-a-byoc-environment). For more information, please refer to [this document](https://github.com/risingwavelabs/risingwave-cloud/blob/main/cli/README.md#installation).
```

Comment:
`https://github.com/risingwavelabs/risingwave-cloud/blob/main/cli/README.md#installation` is in private repo.